### PR TITLE
updated freeze fix  from caoli updated to 1.13.X

### PR DESCRIPTION
**Pull request by wesleyvanneck**
## note it's not tested yet so don't merge yet

### CODE REVIEW

**Comment by wesleyvanneck** [comment]
Note not tested am gone test later

File: src/protocolsupport/protocol/storage/netcache/PEPositionQueue.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+		Location loc = location;
+		location = null;
+
+		if (loc == _location) {
+			return;
+		}
+
+//        System.out.println("!!! RE SEND LATEST POSITION LOCATION");
+
+		((CraftPlayer) conn.getPlayer()).getHandle().playerConnection.teleport(loc);
```
Comment (Shevchik) [inline src/protocolsupport/protocol/storage/netcache/PEPositionQueue.java]: Async unsafe access (with nms), a packet wrapper factory needs to be used.

Comment (wesleyvanneck) [reply src/protocolsupport/protocol/storage/netcache/PEPositionQueue.java]: How? @Shevchik

Comment (wesleyvanneck) [reply src/protocolsupport/protocol/storage/netcache/PEPositionQueue.java]: @Shevchik  what can happen like this ? I try fix it soon but is it really a problem

**Comment by wesleyvanneck** [comment]
I hope it works but I need find a way to make it Async save

**Comment by wesleyvanneck** [comment]
@7kasper @MrPowerGamerBR  can guys help me if you guys have time

**Pull request closed without merging**